### ISSUE TITLE
tooltip for wrong email

### ISSUE THREAD

**Issue by manhair** [swsnu/swppfall2019#36]
Should the tooltip for wrong email contain under sentences?
![스크린샷, 2019-09-14 22-08-31](https://user-images.githubusercontent.com/43872798/64908518-42c92980-d73c-11e9-99a6-9595e8d61b2a.png)

**Comment by hy00nc** [comment]
No, they are not required. Just the first bullet starting with *characters  other than @* is enough.
